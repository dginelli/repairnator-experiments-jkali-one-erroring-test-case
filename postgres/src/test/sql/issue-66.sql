CREATE SCHEMA issue_66;
CREATE TABLE issue_66.test(id serial8 not null primary key);
CREATE INDEX idxissue_66_test ON issue_66.test USING zombodb (zdb('issue_66.test', ctid), zdb(test)) WITH (url='http://localhost:9200/');

CREATE SCHEMA issue_66_other;
CREATE TABLE issue_66_other.test(id serial8 not null primary key);
CREATE INDEX idxissue_66_test ON issue_66_other.test USING zombodb (zdb('issue_66_other.test', ctid), zdb(test)) WITH (url='http://localhost:9200/');


SELECT zdb_determine_index('issue_66.test')::regclass = 'issue_66.idxissue_66_test'::regclass;
SELECT zdb_determine_index('issue_66_other.test')::regclass = 'issue_66_other.idxissue_66_test'::regclass;


DROP SCHEMA issue_66 CASCADE;
DROP SCHEMA issue_66_other CASCADE;