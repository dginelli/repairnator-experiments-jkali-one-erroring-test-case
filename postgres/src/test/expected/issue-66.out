CREATE SCHEMA issue_66;
CREATE TABLE issue_66.test(id serial8 not null primary key);
CREATE INDEX idxissue_66_test ON issue_66.test USING zombodb (zdb('issue_66.test', ctid), zdb(test)) WITH (url='http://localhost:9200/');
CREATE SCHEMA issue_66_other;
CREATE TABLE issue_66_other.test(id serial8 not null primary key);
CREATE INDEX idxissue_66_test ON issue_66_other.test USING zombodb (zdb('issue_66_other.test', ctid), zdb(test)) WITH (url='http://localhost:9200/');
SELECT zdb_determine_index('issue_66.test')::regclass = 'issue_66.idxissue_66_test'::regclass;
 ?column? 
----------
 t
(1 row)

SELECT zdb_determine_index('issue_66_other.test')::regclass = 'issue_66_other.idxissue_66_test'::regclass;
 ?column? 
----------
 t
(1 row)

DROP SCHEMA issue_66 CASCADE;
NOTICE:  drop cascades to table issue_66.test
DROP SCHEMA issue_66_other CASCADE;
NOTICE:  drop cascades to table issue_66_other.test
